packages/docs/src/components/ComponentThemeOptions.tsx

```diff
@@ -17,12 +17,11 @@ const componentThemes = {
 };
 const masterThemes = { core: CoreTheme, healthcare: HealthcareTheme, medicare: MedicareTheme };
 
-type ThemeNames = 'core' | 'healthcare' | 'medicare';
```

Comment:
In an effort to allow some flexibility for adding future themes without needing to update a bunch of files I just changed this to string as generally references to this type are undefined until runtime.